access/email/Makefile

```
@@ -58,3 +59,9 @@ docker-build: ## Build docker image with the plugin.
 .PHONY: docker-push
 docker-push: 
 	docker push ${DOCKER_IMAGE}
+
+.PHONY: docker-promote
+docker-promote:
+	docker pull ${DOCKER_IMAGE}
+	docker tag ${DOCKER_IMAGE} ${DOCKER_IMAGE_QUAY}
+	docker push ${DOCKER_IMAGE_QUAY}
```

Same for other promote targets below.
```suggestion
	docker pull ${DOCKER_IMAGE} && \
	docker tag ${DOCKER_IMAGE} ${DOCKER_IMAGE_QUAY} && \
	docker push ${DOCKER_IMAGE_QUAY}
```

---

Resolved with https://github.com/gravitational/teleport-plugins/pull/466/commits/2bec230f19840dfcfa1d72fd5740ac736bfb38ab